vocabularies/Common.xml

```diff
@@ -1294,19 +1295,19 @@ Upon modification field control logic is invoked so that meta-information like h
 
       <!-- Metadata annotations that can also appear as instance annotations -->
 
-      <Term Name="CreatedAt" Type="Edm.DateTimeOffset" AppliesTo="EntityType">
+      <Term Name="CreatedAt" Type="Edm.DateTimeOffset" Nullable="false" AppliesTo="EntityType">
```

Comment:
```suggestion
      <Term Name="CreatedAt" Type="Edm.DateTimeOffset" AppliesTo="EntityType">
```
In the stateful SD OData service, before an instance is saved, it has CreatedAt = null.

Reply:
Ok, explicitly nullable now